tests/lightning_tensor/test_quimb_mps.py

```diff
@@ -30,6 +35,105 @@
 if LightningDevice._CPP_BINARY_AVAILABLE:
     pytest.skip("Device doesn't have C++ support yet.", allow_module_level=True)
 
+THETA = np.linspace(0.11, 1, 3)
+PHI = np.linspace(0.32, 1, 3)
+
+# gates for which interface support is tested
+ops = {
+    "Identity": qml.Identity(wires=[0]),
+    "BlockEncode": qml.BlockEncode([[0.1, 0.2], [0.3, 0.4]], wires=[0, 1]),
+    "CNOT": qml.CNOT(wires=[0, 1]),
+    "CRX": qml.CRX(0, wires=[0, 1]),
+    "CRY": qml.CRY(0, wires=[0, 1]),
+    "CRZ": qml.CRZ(0, wires=[0, 1]),
+    "CRot": qml.CRot(0, 0, 0, wires=[0, 1]),
+    "CSWAP": qml.CSWAP(wires=[0, 1, 2]),
+    "CZ": qml.CZ(wires=[0, 1]),
+    "CCZ": qml.CCZ(wires=[0, 1, 2]),
+    "CY": qml.CY(wires=[0, 1]),
+    "CH": qml.CH(wires=[0, 1]),
+    "DiagonalQubitUnitary": qml.DiagonalQubitUnitary(np.array([1, 1]), wires=[0]),
+    "Hadamard": qml.Hadamard(wires=[0]),
+    "MultiRZ": qml.MultiRZ(0, wires=[0]),
+    "PauliX": qml.X(0),
+    "PauliY": qml.Y(0),
+    "PauliZ": qml.Z(0),
+    "X": qml.X([0]),
+    "Y": qml.Y([0]),
+    "Z": qml.Z([0]),
+    "PhaseShift": qml.PhaseShift(0, wires=[0]),
+    "PCPhase": qml.PCPhase(0, 1, wires=[0, 1]),
+    "ControlledPhaseShift": qml.ControlledPhaseShift(0, wires=[0, 1]),
+    "CPhaseShift00": qml.CPhaseShift00(0, wires=[0, 1]),
+    "CPhaseShift01": qml.CPhaseShift01(0, wires=[0, 1]),
+    "CPhaseShift10": qml.CPhaseShift10(0, wires=[0, 1]),
+    "QubitUnitary": qml.QubitUnitary(np.eye(2), wires=[0]),
+    "SpecialUnitary": qml.SpecialUnitary(np.array([0.2, -0.1, 2.3]), wires=1),
+    "ControlledQubitUnitary": qml.ControlledQubitUnitary(np.eye(2), control_wires=[1], wires=[0]),
+    "MultiControlledX": qml.MultiControlledX(wires=[1, 2, 0]),
+    "IntegerComparator": qml.IntegerComparator(1, geq=True, wires=[0, 1, 2]),
+    "RX": qml.RX(0, wires=[0]),
+    "RY": qml.RY(0, wires=[0]),
+    "RZ": qml.RZ(0, wires=[0]),
+    "Rot": qml.Rot(0, 0, 0, wires=[0]),
+    "S": qml.S(wires=[0]),
+    "Adjoint(S)": qml.adjoint(qml.S(wires=[0])),
+    "SWAP": qml.SWAP(wires=[0, 1]),
+    "ISWAP": qml.ISWAP(wires=[0, 1]),
+    "PSWAP": qml.PSWAP(0, wires=[0, 1]),
+    "ECR": qml.ECR(wires=[0, 1]),
+    "Adjoint(ISWAP)": qml.adjoint(qml.ISWAP(wires=[0, 1])),
+    "T": qml.T(wires=[0]),
+    "Adjoint(T)": qml.adjoint(qml.T(wires=[0])),
+    "SX": qml.SX(wires=[0]),
+    "Adjoint(SX)": qml.adjoint(qml.SX(wires=[0])),
+    "Toffoli": qml.Toffoli(wires=[0, 1, 2]),
```

Comment:
As discussed last time,  are we going to add tests for the scenarios that target wires are not adjacent?

Reply:
This has been extensively tested in separate notebooks, although I reported just a subset of those tests here since they require some time to be completed. I plan to add more of them over the next few weeks, possibly after the test architecture has been updated